swap/swap.go

```diff
@@ -157,8 +157,9 @@ func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 	if !exists {
 		return fmt.Errorf("peer %v does not exist", peer.ID())
 	}
-	if balance >= s.disconnectThreshold {
-		return fmt.Errorf("balance for peer %s is over the disconnect threshold %d, disconnecting", peer.ID().String(), s.disconnectThreshold)
+	disconnectThreshold := int64(s.disconnectThreshold)
+	if balance >= disconnectThreshold {
```

Comment:
Do we need this extra variable? We can just `if  balance >= int64(s.disconnectThreshold)`?

Reply:
Changed